amplify/backend/function/neptuneDelete/src/index.py

```diff
@@ -0,0 +1,28 @@
+import json
+import os
+import gremlin_python
+from gremlin_python.driver import client, serializer
+
+def lambda_handler(event, context):
+    endpoint = os.environ['NEPTUNE_ENDPOINT']
+    port = os.environ['NEPTUNE_PORT']
+
+    # Connect to Neptune
+    db = client.Client(f"wss://{endpoint}:{port}/gremlin", "g")
+
+    username = event["pathParameters"]["userId"]
+    query = f"g.V().hasLabel('user').has('username', '{username}').drop()"
+
+    try:
+        db.submit(query)
+        result = "Query executed successfully"
+        statusCode = 200
+    except:
+        result = "Query failed"
+        statusCode = 400
```

Comment:
This is currently a catch-all for a variety of failures modes. Consider extrapolating the exception handling to handle more specific errors. May be too much work for this change, if so add a task to Linear and link to this comment.

Examples:
403 - User does not have permission
404 - Resource not found
410 - Resource already deleted